### PR TITLE
Feature add: update script

### PR DESCRIPTION
**Pull request by CarlosLoboxyz**
This snipet will allow you to update your ani-cli instance automatically
for _**future releases only**_

It's pretty handy for people who have their ani-cli inside `.local/bin` or something like that

Edit: Someone could say
>You could just symlink ani-cli from the git repo

But i think this make the script more usefull for people who only want to update without messing with git

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +46,8 @@ A cli to browse and watch anime. This tool scrapes the site [gogoanime](https://
 ```
 grep
 curl
+diff
```
Comment (Dink4n) [inline README.md:49]: Can you move these dependencies to optional

**Review by Dink4n — APPROVED** [review]

**Comment by ura43** [comment]
We had our fun with symlinking and git update checking. Most fragile thing ever. This is better for sure

**Comment by CoolnsX** [comment]
Yup.. looks pretty solid to me...

**Review by ura43 — APPROVED** [review]

**Comment by ura43** [comment]
Good work, merging now

**Pull request merged**